### PR TITLE
fix: resolve eslint dependencies for proper watching

### LINKED ISSUE #580: eslint reports errors from files excluded from tsconfig.json - reproducing in 6.2, didn't happen in 5.x
## Current behavior
**In version 6.2.0**: ESLint will report errors from files that are excluded in tsconfig.json

## Expected behavior
**In version 5.2.0**: ESLint does NOT report errors from files that are excluded in tsconfig.json

## Steps to reproduce the issue
`npm run dev`, then change fork-ts-checker-webpack-plugin version from 6.2 to 5.2 to see it fixed

## Issue reproduction repository
https://github.com/grumd/fork-ts-checker-webpack-plugin-issue

## Screenshots:
image
image

There doesn't seem to be an option to exclude files in plugin's eslint configuration, and 5.x versions worked differently from now, seems like a regression bug.

### PR DESCRIPTION
**Pull request by piotr-oles**
✅ Closes: #580

### CODE REVIEW

**Review by johnnyreilly — APPROVED** [review]

**Comment by uccmen** [comment]
would love to see this PR gets merged soon!

**Comment by piotr-oles** [comment]
yep, unfortunately I have some issues with Windows (as always) :D

File: src/eslint-reporter/reporter/EsLintReporter.ts [hunk 1]
```diff
@@ -3,54 +3,108 @@ import { CLIEngine, LintReport, LintResult } from '../types/eslint';
...
+              });
+            })
+        )
+      );
+
+      for (const resolvedGlob of resolvedGlobs) {
+        for (const resolvedFile of resolvedGlob) {
+          if (isFileIncluded(resolvedFile)) {
+            includedFiles.add(resolvedFile);
+          }
```
Comment (ratherblue) [inline src/eslint-reporter/reporter/EsLintReporter.ts:53]: This introduced a breaking change, see #606

**Pull request merged**